src/commands/acl-save.json

```diff
@@ -0,0 +1,23 @@
+{
+    "SAVE": {
+        "summary": "Save the current ACL rules in the configured ACL file",
+        "complexity": "O(N). Where N is the number of configured users.",
+        "group": "server",
+        "since": "6.0.0",
+        "arity": 2,
+        "container": "ACL",
+        "function": "aclCommand",
+        "command_flags": [
+            "admin",
+            "noscript",
+            "loading",
+            "stale",
+            "sentinel"
+        ],
+        "acl_categories": [
+            "admin",
+            "slow",
+            "dangerous"
+        ]
+    }
+}
```

Comment:
it's not possible to add a `,` after the last element in an array in JSON, right?
it sucks since adding another element will change two lines.
p.s. maybe we should add a newline at the end of file, some tools give an annoying silly warning for that.